canPlayerAccess compile errors

I think the change due to #1251 caused some compiling issues:

```
./minecraft/ArsMagica2/build/sources/java/am2/blocks/BlockKeystoneReceptacle.java:56: error: method canPlayerAccess in class KeystoneUtilities cannot be applied to given types;
                        if (!par1World.isRemote && KeystoneUtilities.instance.canPlayerAccess(receptacle, par5EntityPlayer, KeystoneAccessType.USE)){
                                                                             ^
  required: IKeystoneLockable,EntityPlayer
  found: TileEntityKeystoneRecepticle,EntityPlayer,KeystoneAccessType
  reason: actual and formal argument lists differ in length
./minecraft/ArsMagica2/build/sources/java/am2/blocks/BlockKeystoneChest.java:61: error: method canPlayerAccess in class KeystoneUtilities cannot be applied to given types;
                        if (!KeystoneUtilities.instance.canPlayerAccess(te, entityplayer, KeystoneAccessType.USE)){
                                                       ^
  required: IKeystoneLockable,EntityPlayer
  found: TileEntityKeystoneChest,EntityPlayer,KeystoneAccessType
  reason: actual and formal argument lists differ in length
./minecraft/ArsMagica2/build/sources/java/am2/blocks/BlockKeystoneChest.java:81: error: method canPlayerAccess in class KeystoneUtilities cannot be applied to given types;
                if (KeystoneUtilities.instance.canPlayerAccess(receptacle, player, KeystoneAccessType.BREAK)){
                                              ^
  required: IKeystoneLockable,EntityPlayer
  found: TileEntityKeystoneChest,EntityPlayer,KeystoneAccessType
  reason: actual and formal argument lists differ in length
./minecraft/ArsMagica2/build/sources/java/am2/blocks/BlockKeystoneChest.java:160: error: method canPlayerAccess in class KeystoneUtilities cannot be applied to given types;
                if (!KeystoneUtilities.instance.canPlayerAccess(chest, player, KeystoneAccessType.BREAK)) return false;
                                               ^
  required: IKeystoneLockable,EntityPlayer
  found: TileEntityKeystoneChest,EntityPlayer,KeystoneAccessType
  reason: actual and formal argument lists differ in length
./minecraft/ArsMagica2/build/sources/java/am2/blocks/BlockSummoner.java:52: error: method canPlayerAccess in class KeystoneUtilities cannot be applied to given types;
                        if (KeystoneUtilities.instance.canPlayerAccess((IKeystoneLockable)par1World.getTileEntity(par2, par3, par4), par5EntityPlayer, KeystoneAccessType.USE)){
                                                      ^
  required: IKeystoneLockable,EntityPlayer
  found: IKeystoneLockable,EntityPlayer,KeystoneAccessType
  reason: actual and formal argument lists differ in length
./minecraft/ArsMagica2/build/sources/java/am2/blocks/BlockSummoner.java:133: error: method canPlayerAccess in class KeystoneUtilities cannot be applied to given types;
                if (!KeystoneUtilities.instance.canPlayerAccess(lockable, player, KeystoneAccessType.BREAK)) return false;
                                               ^
  required: IKeystoneLockable,EntityPlayer
  found: IKeystoneLockable,EntityPlayer,KeystoneAccessType
  reason: actual and formal argument lists differ in length
./minecraft/ArsMagica2/build/sources/java/am2/blocks/BlockMagiciansWorkbench.java:86: error: method canPlayerAccess in class KeystoneUtilities cannot be applied to given types;
                        if (KeystoneUtilities.instance.canPlayerAccess((IKeystoneLockable)te, player, KeystoneAccessType.USE)){
                                                      ^
  required: IKeystoneLockable,EntityPlayer
  found: IKeystoneLockable,EntityPlayer,KeystoneAccessType
  reason: actual and formal argument lists differ in length
./minecraft/ArsMagica2/build/sources/java/am2/blocks/BlockMagiciansWorkbench.java:125: error: method canPlayerAccess in class KeystoneUtilities cannot be applied to given types;
          if (KeystoneUtilities.instance.canPlayerAccess(receptacle, player, KeystoneAccessType.BREAK)){
                                        ^
  required: IKeystoneLockable,EntityPlayer
  found: TileEntityMagiciansWorkbench,EntityPlayer,KeystoneAccessType
  reason: actual and formal argument lists differ in length
./minecraft/ArsMagica2/build/sources/java/am2/blocks/BlockMagiciansWorkbench.java:192: error: method canPlayerAccess in class KeystoneUtilities cannot be applied to given types;
                if (!KeystoneUtilities.instance.canPlayerAccess(lockable, player, KeystoneAccessType.BREAK)) return false;
                                               ^
```

concluding with
```
./minecraft/ArsMagica2/build/sources/java/am2/utility/KeystoneUtilities.java:16: error: KeystoneUtilities is not abstract and does not override abstract method canPlayerAccess(IKeystoneLockable,EntityPlayer,KeystoneAccessType) in IKeystoneHelper
public class KeystoneUtilities implements IKeystoneHelper{
       ^
./minecraft/ArsMagica2/build/sources/java/am2/utility/KeystoneUtilities.java:77: error: method does not override or implement a method from a supertype
        @Override
        ^
```

I don't know the intention behind the changes, do I am not sure which one is the proper way.

--- a/src/main/java/am2/utility/KeystoneUtilities.java
+++ b/src/main/java/am2/utility/KeystoneUtilities.java
@@ -2,6 +2,7 @@ package am2.utility;
 
 import am2.api.blocks.IKeystoneLockable;
 import am2.api.items.IKeystoneHelper;
+import am2.api.items.KeystoneAccessType;
 import am2.items.ItemKeystone;
 import am2.items.ItemRune;
 import am2.items.ItemsCommonProxy;
@@ -10,10 +11,17 @@ import net.minecraft.entity.EntityLivingBase;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.ChatComponentText;
+import net.minecraft.util.StatCollector;
 
 import java.util.ArrayList;
 
 public class KeystoneUtilities implements IKeystoneHelper{
+  
+	// constants to pass to the canPlayerAccess() function to say what you're trying to do
+	// please use these by reference instead of hard-coded numbers
+	public static final int MODE_NONE = 0;
+	public static final int MODE_USE = 1;
+	public static final int MODE_BREAK = 2;
 
 	public static final KeystoneUtilities instance = new KeystoneUtilities();
 
@@ -29,7 +37,7 @@ public class KeystoneUtilities implements IKeystoneHelper{
 				}
 				player.addChatMessage(new ChatComponentText(combo));
 			}else{
-				player.addChatMessage(new ChatComponentText("No Key Present."));
+				player.addChatMessage(new ChatComponentText(StatCollector.translateToLocal("am2.tooltip.noKeyPresent")));
 			}
 			ExtendedProperties.For(player).isRecoveringKeystone = false;
 			return true;
@@ -75,7 +83,7 @@ public class KeystoneUtilities implements IKeystoneHelper{
 
 
 	@Override
-	public boolean canPlayerAccess(IKeystoneLockable inventory, EntityPlayer player){
+	public boolean canPlayerAccess(IKeystoneLockable inventory, EntityPlayer player, KeystoneAccessType accessMode){
 		ItemStack[] runes = inventory.getRunesInKey();
 		long key = getKeyFromRunes(runes);
 
@@ -104,6 +112,12 @@ public class KeystoneUtilities implements IKeystoneHelper{
 			}
 		}
 
+		if (accessMode == KeystoneAccessType.USE){
+			player.addChatMessage(new ChatComponentText(StatCollector.translateToLocal("am2.tooltip.wrongKeystoneUse")));
+		}
+		else if (accessMode == KeystoneAccessType.BREAK){
+			player.addChatMessage(new ChatComponentText(StatCollector.translateToLocal("am2.tooltip.wrongKeystoneBreak")));
+		}
 		return false;
 	}
 

